### DISCUSSION TITLE
Looking for list of Xbox controllers that work with PyBricks

### DISCUSSION THREAD

**Discussion by mabba18** [pybricks/support#1480] [Q&A] [answered]
Is there a list of exactly what controllers work?

I have no experience with Xbox, and am looking to buy a controller. [Looking at this page](https://en.wikipedia.org/wiki/Xbox_Wireless_Controller#Summary), it seems that all the controllers after 2016 that have Bluetooth built in (model 1708 and onward) have been updated to use BLE.

Just want to confirm before buying anything. Thanks!

**Comment by laurensvalk** [comment, marked as answer]
You can find information about supported models and the pairing process in the documentation page for the [`XboxController` class](https://docs.pybricks.com/en/latest/iodevices/xboxcontroller.html).

If you're using block coding, you can also right click on a block and choose `help` to easily jump to the relevant documentation in the app.

![image](https://github.com/pybricks/support/assets/12326241/12a3f34a-2f77-4106-b7f7-3d08e001e025)
